SwiftSampleProject/SwiftyVokoderTests/ManagedObjectContextTests.swift

```diff
@@ -75,7 +75,7 @@ class ManagedObjectContextTests: XCTestCase {
 
         self.manager.deleteAllObjects(of: Station.self, context: nil)
         
-        self.expectation(forNotification: NSNotification.Name.NSManagedObjectContextDidSave.rawValue,
+        self.expectation(forNotification: NSNotification.Name(rawValue: NSNotification.Name.NSManagedObjectContextDidSave.rawValue),
```

Comment:
Could this be `self.expectation(forNotification: .NSManagedObjectContextDidSave)`?